files/default/autolock.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-Exec=/usr/bin/autolock.sh
+Exec=sh -c "$HOME/bin/autolock.sh"
```

Comment:
I do not think putting executable files in users HOME is a good idea.
I would choose one of these solutions:

1. Place a generic autolock.sh script in /usr/bin, and a per-user configuration file in userse HOME
2. Avoid autolock.sh. Justs put the right xautolock parameters in the .desktop file of every user.

Reply:
Option 1 implemented.